docs/themes.md

```diff
@@ -28,13 +28,32 @@ OPTIONS
   --port=port      [default: 4567] Port for the http server to use
   --logs           Tail logs
   --no-livereload  Disable live-reloading the preview when a change is made
+  --ssl-cert       SSL Certificate used to start the server in HTTPS mode
+  --ssl-key        SSL Key used to start the server in HTTPS mode
```

Comment:
Nit: checked webpack and vite docs and they name these options `https` instead of `ssl`. Could make sense to align.

Reply:
Makes sense. Also, SSL is not completely correct, since now it is called TLS